### ISSUE TITLE
Cleanup form-object

### ISSUE THREAD

**Issue by jotoeri** [nextcloud/forms#290]
- Create proper object
- Cleanup `form.form` to just `form`
- Remove computed form-variables expired&expires
- Which properties will be necessary for forms list, and which ones suffice to be loaded when editing the form?

Just simple Data from DB:
```
form: {
	*id: (int)
	*hash: "string"
	*title: "string"
	description: "string"
	ownerId: "string"
	access: {
		type: "(string)|public|registered|selected|"
		users: ["(string)uid1", "(string)uid2"]
		groups: []
	}
	created: timestamp
	*expires: timestamp
	anonymous: bool
	submitOnce: bool
	questions: []
}
```
- *Necessary for forms-List: `id`/`hash`, `title`, `expires`?
- Should be done after #256 as it should be consistent?
- @skjnldsv Do you still want to group some parts?

**Comment by skjnldsv** [comment]
> * @skjnldsv Do you still want to group some parts?

Yes, that would be cleaner :)

**Comment by skjnldsv** [comment]
Fixed with upcomming 2.0
